spring-cloud-kubernetes-client-autoconfig/src/main/java/org/springframework/cloud/kubernetes/client/KubernetesClientHealthIndicator.java

```diff
@@ -16,43 +16,44 @@
 
 package org.springframework.cloud.kubernetes.client;
 
-import java.util.HashMap;
+import java.util.Collections;
 import java.util.Map;
 
 import io.kubernetes.client.openapi.models.V1Pod;
 
 import org.springframework.cloud.kubernetes.commons.AbstractKubernetesHealthIndicator;
 import org.springframework.cloud.kubernetes.commons.PodUtils;
+import org.springframework.util.CollectionUtils;
 
 /**
  * @author Ryan Baxter
  */
 public class KubernetesClientHealthIndicator extends AbstractKubernetesHealthIndicator {
 
```

Comment:
minor things, `Collections::singletonMap`, `final`, move things "logically" together for easier read of the code